packages/paste-core/components/disclosure/stories/index.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs, text, select} from '@storybook/addon-knobs';
+import {HeadingProps} from '@twilio-paste/heading';
+import {Stack} from '@twilio-paste/stack';
+import {Disclosure, DisclosureHeading, DisclosureContent, Variants} from '../src';
+
+const headingVariantOptions = ['heading10', 'heading20', 'heading30', 'heading40', 'heading50', 'heading60'];
+
+export const ExampleDisclosures: React.FC<{
+  disabled?: boolean;
+  variant?: Variants;
+  headingVariant: '10' | '20' | '30' | '40' | '50' | '60';
+}> = props => {
+  return (
+    <Stack orientation="vertical" spacing="space70">
+      <Disclosure visible variant={props.variant}>
+        <DisclosureHeading
+          as="h2"
+          disabled={props.disabled}
+          variant={`heading${props.headingVariant}` as HeadingProps['variant']}
+        >
+          Heading variant {props.headingVariant}
+        </DisclosureHeading>
+        <DisclosureContent>Disclosure content</DisclosureContent>
+      </Disclosure>
+      <Disclosure variant={props.variant}>
+        <DisclosureHeading
+          as="h2"
+          disabled={props.disabled}
+          variant={`heading${props.headingVariant}` as HeadingProps['variant']}
+        >
+          Heading variant {props.headingVariant}
+        </DisclosureHeading>
+        <DisclosureContent>Disclosure content</DisclosureContent>
+      </Disclosure>
+    </Stack>
+  );
+};
+
+storiesOf('Components|Disclosure', module)
+  .addDecorator(withKnobs)
+  .add('All variants', () => {
+    const asOptions = text('as', 'h2') as HeadingProps['as'];
+    const headingVariantValue = select(
+      'heading variant',
+      headingVariantOptions,
+      'heading10'
+    ) as HeadingProps['variant'];
```

Comment:
This is passing the correct variant for `headingVariant` (i.e.: "heading10" "heading30" ...), but the next examples are passing a number string value (i.e.: "10" "30") and string concatenating.  I think this "all variants" story is broken (haven't ran locally).

Can you just update the story to not do anything fancy? So pass like `headingVariant="heading20"`, update the typing, and remove the string concat? Makes it much easier to follow.

Reply:
> I think this "all variants" story is broken (haven't ran locally)

It works. I'll update the `ExampleDisclosures` component though